script/NestedSignFromJson.s.sol

```diff
@@ -36,25 +36,4 @@ contract NestedSignFromJson is NestedMultisigBuilder, JsonTxBuilderBase {
     function _ownerSafe() internal view override returns (address) {
         return vm.envAddress("OWNER_SAFE");
     }
-
-    function _postCheck(Vm.AccountAccess[] memory accesses, SimulationPayload memory simPayload)
-        internal
-        virtual
-        override
-    {
-        if (msg.sig == this.approveJson.selector) {
-            console.log("Skipping assertions on the approval call");
-            return;
-        }
-        _nestedPostCheck(accesses, simPayload);
-    }
-
-    function _nestedPostCheck(Vm.AccountAccess[] memory accesses, SimulationPayload memory simPayload)
-        internal
-        virtual
-    {
-        accesses; // Silences compiler warnings.
-        simPayload;
-        require(false, "_nestedPostCheck not implemented");
-    }
```

Comment:
marking this contract as `abstract` achieves the same goal (sub-contracts must implement `_postCheck`)